weblab/static/js/compare.js

```diff
@@ -284,7 +262,8 @@ function parseEntities (entityObj)
 				var file = entity.files[j],
 					sig = file.name.hashCode();
 				file.signature = sig;
-				file.url = contextPath + "/download/" + entityType.charAt(0) + "/" + convertForURL (entity.name) + "/" + entity.id + "/" + file.id + "/" + convertForURL (file.name);
+				file.url = file.url;
```

Comment:
Probably don't need this line? :)

Reply:
Looks good, apart from this fix is missing :)

Reply:
d'oh! Fixed.